kinetic/distribution.yaml

```diff
@@ -4083,6 +4083,14 @@ repositories:
       url: https://github.com/ros/ros_tutorials.git
       version: kinetic-devel
     status: maintained
+  ros_type_introspection:
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/facontidavide/ros_type_introspection-release.git
+      version: 0.2.0-0
+    status: developed
+    status_description: Fully funtional implementation, but more testing is needed.
```

Comment:
The status description is only used to communicate the reason of the status is end-of-life. It shouldn't be set of the status is developed.